apis/ocpdiag/README.md

```diff
@@ -173,5 +215,6 @@ document describes how to define and use the OCPDiag Hardware Interface.
 
 ## Contact information
 
-**Team:** ocp-test-validation@OCP-All.groups.io
```

Comment:
Keep the old email

Reply:
Done. Also removed some info only relevant to google binaries